packages/core/src/components/Pagination/index.tsx

```diff
@@ -37,20 +44,28 @@ export type Props = {
 /** Pagination controls. */
 export class Pagination extends React.Component<Props & WithStylesProps> {
   static defaultProps = {
+    centerAlign: false,
+    endAlign: false,
     fetching: false,
     hasNext: false,
     hasPrev: false,
     showBookends: false,
+    startAlign: false,
   };
 
   static propTypes = {
+    centerAlign: alignProp,
+    endAlign: alignProp,
+    startAlign: alignProp,
```

Comment:
small nit around alphabetizing to match defined types and defaultProps for consistency